modules/platforms/cpp/ignite/schema/binary_tuple_builder.h

```diff
@@ -258,6 +314,68 @@ class binary_tuple_builder {
      */
     void append_uuid(uuid value) { put_uuid(value); }
 
+    /**
+     * @brief Appends a big integer for the next element.
+     *
+     * @param value Element value.
+     */
+    void append_number(const big_integer &value) { put_number(value); }
```

Comment:
Now I think there is no much point to have both `put_` and `append_` methods  essentially the same. We should lift the code from `put_` methods with specific type to corresponding `append_` methods.

Reply:
Done